packages/hURL/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=hURL
+pkgver=20.afca9c5
+pkgrel=1
+pkgdesc='Hexadecimal & URL (en/de)coder.'
+arch=('any')
+groups=('blackarch' 'blackarch-misc' 'blackarch-exploitation')
+url='https://github.com/fnord0/hURL'
+license=('custom:unknown')
+depends=('perl' 'perl-cgi' 'perl-uri' 'perl-digest-sha1' 'perl-mime-types')
+makedepends=('git')
+source=("git+https://github.com/fnord0/$pkgname.git")
+sha512sums=('SKIP')
+
+prepare() {
+  cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+build() {
+  cd $pkgname
+}
+
+package() {
+  cd $pkgname
+
+  install -Dm 755 "$srcdir/$pkgname/$pkgname" "$pkgdir/usr/bin/$pkgname"
```

Comment:
`install -Dm 755 fURL "$pkgdir/usr/bin/$pkgname"`

Reply:
adjusted / not committed yet